RELEASE.md

```diff
@@ -14,9 +14,11 @@ We maintain a separate branch for each minor release, named `release-<major>.<mi
 
 Note that branch protection kicks in automatically for any branches whose name starts with `release-`. Never use names starting with `release-` for branches that are not release branches.
 
-The usual flow is to merge new features and changes into the main branch and to merge bug fixes into the latest release branch. Bug fixes are then merged into main from the latest release branch. The main branch should always contain all commits from the latest release branch. As long as main hasn't deviated from the release branch, new commits can also go to main, followed by merging main back into the release branch.
-
-If a bug fix got accidentally merged into main after non-bug-fix changes in main, the bug-fix commits have to be cherry-picked into the release branch, which then have to be merged back into main. Try to avoid that situation.
+- Every new feature and change should be merged into the main branch with the exception of bug fixes. This can be managed in two different ways: 
+   - Bug fixes that need to exist on the release branches should be merged to main and backported to the release branch.
+   - Bug fixes that need not exist on main should be merged to the release branch only. 
```

Comment:
What happens when we merge said branch back into main, or are we scrapping that workflow

Reply:
Yes, we should scrap that workflow. Just cherry-pick what's necessary.